recipes-example/template-dbus-service/template-dbus-service.bb

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2019 Luxoft Sweden AB
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+
+DESCRIPTION = "Template for creating a D-Bus Service"
+HOMEPAGE = "https://github.com/Pelagicore/template-dbus-service"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+PV = "1.0+git${SRCREV}"
+PR = "r0"
+
+SRC_URI = "git@github.com:Pelagicore/template-dbus-service.git;protocol=ssh;branch=master"
```

Comment:
https one will be much more readable

Reply:
Not that big of a difference, I would say, also we have the same in all the other recipes in this repo, so trying to keep to the format. Do you have a strong opinion on this?

Reply:
No, but `https` is shorter, more readable and works in networks where port `22` is blocked.

Reply:
It would be the same exact length.
In my opinion we should use git unless we are having issues with it, which we are not. And as far as I can see git is the most used way to fetch the source for all our external layers. Not saying we should do as everyone else, just saying. The biggest reason not to change is to keep it uniform and that we have not had issues with this way so far.

Reply:
> It would be the same exact length

Nope, it would be `https://github.com/Pelagicore/template-dbus-service`

> we should use git unless we are having issues with it, which we are not.

We are not, but some people even have news websites blocked in by their corporate firewalls.

> The biggest reason not to change is to keep it uniform

ihmo, it's time to change it everywhere

Reply:
Look what I found: https://github.com/Pelagicore/meta-bistro/issues/131

Reply:
Hum... (pwd=pelux/sources):
```bash
$ grep -rI SRC_URI.*git:// . | grep -v protocol=https | wc -l
406
```

Reply:
So is it the protocol we are discussing or the url? Or both?
@martin-ejdestig Your grep looks like it would find the git URI using https protocol.

Reply:
So there they changed it to 
`git://github.com/profusion/lightmediascanner.git;protocol=https`

Reply:
Alright. After the discussion I will change to use https in this recipe.

Reply:
https://help.github.com/en/articles/which-remote-url-should-i-use#cloning-with-https-urls-recommended